core/src/main/scala/org/ensime/core/RichPresentationCompiler.scala

```diff
@@ -145,17 +145,28 @@ trait RichCompilerControl extends CompilerControl with RefactoringControl with C
   def askReloadAndTypeFiles(files: Iterable[SourceFile]) =
     askOption(reloadAndTypeFiles(files))
 
-  def askUsesOfSymAtPos(pos: Position)(implicit ec: ExecutionContext): Future[List[RangePosition]] = {
+  def askUsesOfSymAtPos(pos: Position)(implicit ec: ExecutionContext): Future[List[LineSourcePosition]] = {
```

Comment:
we want to move this query into the `SearchService`, shouldn't need the presentation compiler at all